deploy/route_crd.yaml

```diff
@@ -13,28 +13,29 @@ spec:
       served: true
       # One and only one version must be marked as the storage version.
       storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
+      additionalPrinterColumns:
+        - name: Host
+          type: string
+          jsonPath: .status.ingress[0].host
+        - name: Admitted
+          type: string
+          jsonPath: .status.ingress[0].conditions[?(@.type=="Admitted")].status
+        - name: Service
+          type: string
+          jsonPath: .spec.to.name
+        - name: TLS
+          type: string
+          jsonPath: .spec.tls.type
   # either Namespaced or Cluster
   scope: Namespaced
-  subresources:
-    # enable spec/status
-    status: {}
```

Comment:
Why is `subresources:` deleted?  The CRD should still define a `status` subresource.

Reply:
Correct, I have removed it cluelessly. It's back